regen-network/regen-web
# fix: issues with Wallet Connect

## Description

Closes: #2263



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



From https://deploy-preview-2269--regen-marketplace.netlify.app/, try to connect using Wallet Connect from desktop and mobile (don't expect tx signing to work since redwood is not available on Keplr mobile - only regen mainnet and deploy previews use redwood)

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
